front-end/src/components/GithubFeedItem.tsx

```diff
@@ -10,11 +10,16 @@ interface Props {
  * @param props
  */
 export default function LikeListItem(props: Props) {
```

Comment:
I think this should be `GithubFeedItem`?

Reply:
Fixed